.github/workflows/testing.yml

```diff
@@ -22,3 +22,30 @@ jobs:
         run: bin/test_schema
       - name: Run registrar validation
         run: bin/test_registrar
+
+  ui:
+    name: UI Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2.3.4
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '15'
+      - name: Install Chrome
+        run: |
+          wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+          sudo apt install -y ./google-chrome-stable_current_amd64.deb
+      - name: Install dependencies
+        run: |
```

Comment:
consistency with other lines -- cd udms on same line... (true for the next one as well)